tests/lib/boot.sh

```diff
@@ -33,3 +33,15 @@ bootenv_unset() {
         fw_setenv "$var"
     fi
 }
+
+get_boot_path() {
+    if [ -f /boot/uboot/uboot.env ]; then
+        echo "/boot/uboot/"
+    elif [ -f /boot/grub/grubenv ]; then
+        echo "/boot/grub/"
+    else
+        echo "Cannot determine bootdir in /boot:"
```

Comment:
Nitpick: the function talks about boot_path, could we say something like "cannot determine boot path?", also ideally this would go to stderr not to be somehow confused with normal output. The same goes for the directory listing below.

Reply:
done